metadata.yml

```
@@ -27,8 +27,8 @@ annotations:
   io.kubewarden.policy.version: 1.1.1
   io.kubewarden.policy.description: A policy that detects webhook services used by admission controllers that are accidentally exposed outside of the cluster
   io.kubewarden.policy.author: Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>
-  io.kubewarden.policy.url: https://github.com/yourorg/do-not-expose-admission-controller-webhook-services
-  io.kubewarden.policy.source: https://github.com/yourorg/do-not-expose-admission-controller-webhook-services
+  io.kubewarden.policy.url: https://github.com/kubewarden/do-not-expose-admission-controller-webhook-services
```

Eh, that's a long name :)
```suggestion
  io.kubewarden.policy.url: https://github.com/kubewarden/do-not-expose-admission-controller-webhook-services-policy
```